tests/unit/utils/test_boto3mod.py

```diff
@@ -0,0 +1,142 @@
+"""
+    Tests for salt.utils.boto3mod
+"""
+
+import random
+import string
+
+import salt.loader
+import salt.utils.boto3mod as boto3mod
+from salt.utils.versions import LooseVersion
+from tests.support.mixins import LoaderModuleMockMixin
+from tests.support.mock import MagicMock, patch
+from tests.support.unit import TestCase, skipIf
+
+try:
+    import boto3
+    from botocore.exceptions import ClientError
+
+    HAS_BOTO3 = True
+except ImportError:
+    HAS_BOTO3 = False
+
+REQUIRED_BOTO3_VERSION = "1.2.1"
+
+
+def __virtual__():
+    """
+    Returns True/False boolean depending on if Boto3 is installed and correct
+    version.
+    """
+    if not HAS_BOTO3:
+        return False
+    if LooseVersion(boto3.__version__) < LooseVersion(REQUIRED_BOTO3_VERSION):
+        return (
+            False,
+            (
+                "The boto3 module must be greater or equal to version {}"
+                "".format(REQUIRED_BOTO3_VERSION)
+            ),
+        )
+    return True
+
+
+REGION = "us-east-1"
```

Comment:
we try to avoid module level variables in tests if we can, because this has caused issues that are hard to find and troubleshoot when other files end up importing these modules and they use the same variable names. One simple approach if you need all of your test cases to access these variables is to add them into the `setUp` funciton and refer to them as `self.region` for example.